R/expect-equality.R

```diff
@@ -48,7 +48,6 @@ NULL
 #' @param ... other values passed to [all.equal()]
 expect_equal <- function(object, expected, ..., info = NULL, label = NULL,
                          expected.label = NULL) {
-
```

Comment:
Ideally styler wouldn't strip this new line. It doesn't matter here, but in general it seems like a bad idea

Reply:
Can we define a rule when to add/keep a newline after an opening brace? Is this just for top-level functions or for blocks in general?

Reply:
I think I generally prefer a newline if the function def span multiple lines as it more clearly distinguishes the body from the args.

Reply:
So should we

* Add a section to the tidyverse style guide to leave one line blank after a multi-line function declaration and
* make styler in general preserving line breaks 
?

Not sure whether for the second bullet, we could also make some blank line formatting optional, but I could not think of anything particularly meaningful. Maybe one can choose to either to not touch blank lines, or to reduce the amount of blank lines to one if there is at least two blank lines next to each other.

Reply:
I'm not sure how consistently I adhere to this style. There's also the question of wrapping multiline formals - we should probably be consistent with other calls, and if all arguments don't fit on one line, each argument gets it's own line.